cmd/component.go

```diff
@@ -68,6 +71,36 @@ var componentSetCmd = &cobra.Command{
 	},
 }
 
+var componentDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe the given application",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		er := args[0]
+		p, _ := application.GetCurrent(client)
+		c := project.GetCurrent(client)
+
+		_, r, _ := component.GetComponentSource(client, er, p, c)
+		q, _ := url.List(client, er, p)
```

Comment:
again, the code is hard to read if you use one letter variable names.
and again, check for errors